src/models/HybridSystem.jl

```diff
@@ -0,0 +1,172 @@
+"""
+Representation of a Hybrid System that collects renewable generation, load, thermal generation
+and storage
+"""
+mutable struct HybridSystem{
+    T <: ThermalGen,
+    L <: ElectricLoad,
+    S <: Storage,
+    R <: RenewableGen,
+} <: StaticInjection
+    name::String
+    available::Bool
+    status::Bool
+    bus::Bus
+    active_power::Float64
+    reactive_power::Float64
+    thermal_unit::T
+    electric_load::L
+    storage::S
+    renewable_unit::R
+    # PCC Data
```

Comment:
If I understand this correctly, the following couple parameters are to specify inverter (PCC here) properties? This seems to me is modeled as a DC-coupled system (i.e., shared inverter for the whole system). If we want to model AC-coupled systems, we'll need inverter properties for each component in the hybrid system. Would this representation be able to model it?

Reply:
If you need to model independent interconnections then the information is stored inside of each component that composes the hybrid system